tests/functional/common.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+import grpc
+import os
+import tritonclient.grpc as grpcclient
+from tritonclient.grpc import service_pb2_grpc
+
+def prepare_inputs(input_name, shape):
+    inputs = []
+    inputs.append(grpcclient.InferInput(input_name, shape, "FP32"))
```

Comment:
The datatype can also be passed from the function argument, or loaded from shape.